### PR TITLE
Filter

### PR DESCRIPTION
**Pull request by EmirKapic**

### CODE REVIEW

File: frontend/src/components/Common/DoubleSlider/styles.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    width: 100%;
+    position: absolute;
+    background-color: #C6C6C6;
+    pointer-events: none;
+}
+
+#fromSlider {
+    height: 0;
+    z-index: 1;
+}
```
Comment (farispoljcic-abh) [inline frontend/src/components/Common/DoubleSlider/styles.css:44]: add newline to EOF

File: frontend/src/pages/SellingProcess/ItemInfo.tsx [hunk 2]
```diff
@@ -34,7 +34,10 @@ export default function ItemInfo(props: ItemInfoProps) {
         onClick={(e) => {
           e.stopPropagation();
           props.updateFields({
-            imagePaths: props.imagePaths.filter((img, indx) => indx !== index),
+            imagePaths: props.imagePaths.filter((img, indx) => {
+              img;
+              return indx !== index;
+            }),
```
Comment (farispoljcic-abh) [inline frontend/src/pages/SellingProcess/ItemInfo.tsx:40]: What does this achieve?

File: frontend/src/pages/Shop/Sidebar/PriceFilter.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
+import BarGraph from "components/Common/BarGraph/BarGraph";
+import Button from "components/Common/Button";
+import DoubleSlider from "components/Common/DoubleSlider/DoubleSlider";
+//import DoubleSlider from "components/Common/DoubleSlider/DoubleSlider";
```
Comment (farispoljcic-abh) [inline frontend/src/pages/Shop/Sidebar/PriceFilter.tsx:4]: No commented out code.

File: backend/src/main/java/com/atlantbh/internship/AuctionApp/projections/MaxMinPrice.java [hunk 4]
```diff
@@ -0,0 +1,6 @@
+package com.atlantbh.internship.AuctionApp.projections;
+
+public interface MaxMinPrice {
```
[the remark this answers is not in the document]
Comment (EmirKapic) [reply backend/src/main/java/com/atlantbh/internship/AuctionApp/projections/MaxMinPrice.java:3]: I don't, and I don't think its good but Spring makes every other way infinitely harder and I don't want to fight the framework.

File: backend/src/main/java/com/atlantbh/internship/AuctionApp/controllers/Products/ProductController.java [hunk 5]
```diff
@@ -58,4 +57,9 @@ public ResponseEntity makeNewProduct(@RequestBody NewProductRequest request) {
     public ResponseEntity getRecommendedProducts() {
         return ResponseEntity.ok().body(productService.recommendedProducts());
     }
+
+    @GetMapping("/price-details")
+    public ResponseEntity getExtraProductInfo(ProductParameters productParameters, @RequestParam(defaultValue = "20") Long numberOfBuckets){
```
Comment (farispoljcic-abh) [inline backend/src/main/java/com/atlantbh/internship/AuctionApp/controllers/Products/ProductController.java:62]: We should rename the method and the returned DTO as well.

File: frontend/src/pages/SellingProcess/ItemInfo.tsx (hunk 2 above)
Comment (farispoljcic-abh) [reply frontend/src/pages/SellingProcess/ItemInfo.tsx:40]: Still curious about this ^

**Review by farispoljcic-abh — APPROVED** [review]

**Pull request merged**